### ISSUE TITLE
Discord bot instantly crashes on start (ran out of shards?)

### ISSUE THREAD

**Issue by D3faIt** [brainboxdotcc/DPP#445]
**Git commit reference**
583ea4f149c5555324af82576aed230e14a54b57

**Describe the bug**
Instantly throws an error: `Discord indicates you cannot start enough sessions to boot this cluster! Cluster startup aborted. Try again later.`

**To Reproduce**
Not completely sure, and it might not be related to DPP at all.
There is 2 threads starting the bot, one with `bot.start(dpp::st_wait);` for handling commands, and one with `bot.start(dpp::st_return);` in a while loop with 120 sec between each loop.
I can only assume the issue is in the loop, maybe it's not deleting the bot instance when the thread is sleeping.

```c++
[[noreturn]]
void checkNewPost(){
    // Wait 12 sec before starting the loop
    sleep(12);

    int sleepSecs = 180;
    while (true){
        if (pause_check){
            sleep(sleepSecs);
            continue;
        }

        dpp::cluster bot(BOT_TOKEN, dpp::i_default_intents | dpp::i_message_content, dpp::i_guild_members);
        Post::checkNew(bot);
        bot.start(dpp::st_return);
        sleep(sleepSecs);
    }
}
```

**System Details:**
 - OS: Arch Linux
 - Discord Client used for testing: Desktop

It has been instantly crashing for the past hour or so

**Comment by D3faIt** [comment]
Turning on logging, reveals `Cluster: 0 of 1000 session starts remaining`.
How would you correctly go about handle this kind of use case?

**Comment by braindigitalis** [comment]
...why are you starting in a while loop?

you've ran your bot out of startups, this is a Discord restriction, there is no workaround. don't start the bot 1000 times per day.

**Comment by braindigitalis** [comment]
another issue I noticed with this is you have the intents in the parameter for number of shards... this value is well over 1000...

**Comment by D3faIt** [comment]
Thank you for responding, and sorry for reporting a issue like this. It was kind of frustrating in the moment it happened, and couldn't find much of any answers online.

> I am not sure what you're trying to achieve?

I'm starting the bot with 2 threads, but assumed the issue was in the 2nd thread. I think I fixed the issue by moving the bot instance before the loop, thank you for giving me guidance.
